docs/manual/wcur2.py

```diff
@@ -1,7 +1,13 @@
-# def get_pref_col(self, (maxcol,)):
-#         return self.cursor_x
-#
-# def move_cursor_to_coords(self, (maxcol,), col, row):
-#     assert row == 0
-#     self.cursor_x = col
-#     return True
+from __future__ import annotations
+
+import urwid
+
+
+class Widget(urwid.ListBox):
+    def get_pref_col(self, size):
+        return self.cursor_x
+
+    def move_cursor_to_coords(self, size, col, row):
+        assert row == 0
+        self.cursor_x = col
+        return True
```

Comment:
looks like in docs/manual/widgets.rst where wdur2.py is referenced we can add a `:lines: 7-` to skip the first 6 lines of this file in the docs https://devopstutodoc.readthedocs.io/en/stable/documentation/doc_generators/sphinx/rest_sphinx/code/literalinclude/literalinclude.html#lines-xxx

Reply:
Done. Validated build.
Had to merge another PR (#424) due to broken documentation build in master (Issue #509)